File: fizzbuzz.cpp
#include <iostream>

using namespace std;

int main(void)
{
	cout << "How many terms should be shown?" << endl;

	int maxTerms = 0;
	cin >> maxTerms;

	for (int i = 1; i <= maxTerms; i++)
	{
		if (i % 3 == 0 && i % 5 == 0)
			cout << "Fizz Buzz, ";
		else if (i % 3 == 0)
			cout << "Fizz, ";
		else if (i % 5 == 0)
			cout << "Buzz, ";
		else
			cout << i << ", ";
	}
	
	return 0;
}

File: fizzbuzz.py
print "How many terms should be displayed?"

maxTerms = int(raw_input())

for i in range(1, maxTerms + 1):
    if (i % 3 == 0 and i % 5 == 0):
        print "Fizz Buzz,",
    elif (i % 3 == 0):
        print "Fizz,",
    elif (i % 5 == 0):
        print "Buzz,",
    else:
        print i , ",",

